Make test and make openlane fail

### Description

While running the installation cmds:

> cd $HOME
> git clone https://github.com/The-OpenROAD-Project/OpenLane
> cd OpenLane
> make
> make test

make test leads to :
> docker: Error response from daemon: unable to find groups for spec openlane-user-1000: bufio.Scanner: token too long.
> make: *** [Makefile:204: test] Error 125

And

>  make openlane

 freezes in one build step : 
![image](https://github.com/The-OpenROAD-Project/OpenLane/assets/37119086/f9b1e010-4cde-4bb7-bfe7-bf4f6f588a00)

Note : make openlane, the following lines have been requested to be added to docker/build.sh

--extra-experimental-features flakes\
--extra-experimental-features nix-command\


> ++ nix build --no-link --print-out-paths --accept-flake-config --option system x86_64-linux --extra-platforms x86_64-linux ..#packages.x86_64-linux.openlane1-docker
> error: experimental Nix feature 'flakes' is disabled; add '--extra-experimental-features flakes' to enable it
> + TARBALL=


### Expected Behavior

Basic test passed

### Environment report

```shell
open_pdks bdc9412b3e468c102d01b7cf6337be06ec6e9c9a
Kernel: Linux v5.15.0-105-generic
Distribution: ubuntu 20.04
Python: v3.8.10 (OK)
Container Engine: docker v20.10.22 (OK)
OpenLane Git Version: 14b94a668289d8dc42bc15beb55ec9b3877b870c
python-venv: INSTALLED
---
PDK Version Verification Status: OK
---
Git Log (Last 3 Commits)

14b94a66 2024-05-08T15:41:43+03:00 Migrate Build System to Match OpenLane 2 (#2115) - Mohamed Gaber -  (HEAD -> master, tag: 2024.05.09, origin/master, origin/HEAD)
337ffbf4 2024-04-21T17:01:05+02:00 Update OpenROAD (#2118) - Mohamed Gaber -  (tag: 2024.04.22)
ad7a11ff 2024-04-03T18:41:00+02:00 Update `libparse` (#2108) - Mohamed Gaber -  (tag: 2024.04.04)
---
Git Remotes

origin	git@github.com:The-OpenROAD-Project/OpenLane (fetch)
origin	git@github.com:The-OpenROAD-Project/OpenLane (push)
```


### Reproduction material

Cloned the latest from master branch, commit : 14b94a668289d8dc42bc15beb55ec9b3877b870c

### Relevant log output

```shell
> docker: Error response from daemon: unable to find groups for spec openlane-user-1000: bufio.Scanner: token too long.
> make: *** [Makefile:204: test] Error 125
```

I still get this following after pulling, this seems to be an issue inside the container, I am new to docker.
make test : 
> docker run --rm -v /home/sofiane:/home/sofiane -v /home/sofiane/OpenLane:/openlane -v /home/sofiane/OpenLane/empty:/openlane/install -v /home/sofiane/.volare:/home/sofiane/.volare -e PDK_ROOT=/home/sofiane/.volare -e PDK=sky130A  --user 1000:1002 -e DISPLAY=:0 -v /tmp/.X11-unix:/tmp/.X11-unix -v /home/sofiane/.Xauthority:/.Xauthority --network host --security-opt seccomp=unconfined efabless/openlane:77eb5ffc00966b47b36e332ccbf42103eb95e974-amd64 sh -c "./flow.tcl -design spm -tag openlane_test -overwrite"
> docker: Error response from daemon: unable to find groups for spec openlane-user-1000: bufio.Scanner: token too long.
> make: *** [Makefile:204: test] Error 125

As for the tutorial : https://www.youtube.com/watch?v=QnJzoJjC7RQ

--- a/docs/source/getting_started/installation/installation_linux.md
+++ b/docs/source/getting_started/installation/installation_linux.md
@@ -23,6 +23,10 @@ supported.
 
 First, install Docker following the steps provided [in this link](https://docs.docker.com/engine/install/).
 
+:::{note}
+Make sure Docker is up to date with the latest release provided by your distribution.
+:::
+
 Test if installation was successful:
 
 ```

--- a/docs/source/getting_started/installation/installation_macos.md
+++ b/docs/source/getting_started/installation/installation_macos.md
@@ -12,5 +12,13 @@ brew install python make
 brew install --cask docker
 ```
 
+If brew and the dependencies are already installed, make sure they are up to date:
+
+```sh
+brew update
+brew upgrade python make
+brew upgrade --cask docker
+```
+
 ```{include} installation_common_section.md
 ```

--- a/docs/source/getting_started/installation/installation_ubuntu.md
+++ b/docs/source/getting_started/installation/installation_ubuntu.md
@@ -13,6 +13,11 @@ Only Ubuntu 20.04 and above are supported.
 Next, install Docker.
 Follow [the instructions provided in the Docker documentation here](https://docs.docker.com/engine/install/ubuntu/) as the steps provided below might be outdated.
 
+:::{note}
+If Docker is already installed, make sure that it is up to date by 
+[following these instructions in the Docker documentation](https://docs.docker.com/engine/install/ubuntu/#upgrade-docker-engine-1)
+:::
+
 :::{warning}
 The steps below might become outdated, it is encouraged to follow the link to the official Docker documentation.
 :::

--- a/docs/source/getting_started/installation/installation_win.md
+++ b/docs/source/getting_started/installation/installation_win.md
@@ -10,6 +10,11 @@ The following is the recommended installation method under Windows. Other virtua
 
 1. Follow [official Microsoft documentation for WSL located here](https://docs.microsoft.com/en-us/windows/wsl/install) to install the WSL 2. Make sure your OS version supports WSL 2.
 2. Follow [official steps to Install Docker Desktop on Windows located here](https://docs.docker.com/desktop/install/windows-install/).
+
+   :::{note}
+   Make sure to update Docker if it is already installed.
+   :::
+
 3. Make sure that `WSL 2 Docker engine` is enabled and `Settings` -> `Resource` -> `WSL Integration` is enabled
 4. Make sure that option `Start Docker Desktop when you login` is enabled in `Docker Desktop` -> `Settings`
 

